DEV_ENV_SETUP.md

```diff
@@ -0,0 +1,69 @@
+# Developer Environment Setup Guide for niceML 🍦
+
+## Overview
+
+👋 Welcome to the development environment setup guide for niceML! This guide will walk you through the process of setting up a development environment for contributing to our open-source project. We use Poetry for managing dependencies, and we recommend using conda environments for a seamless development experience. Let's get your environment up and running smoothly! 🚀
+
+## Prerequisites
+
+Before you begin, ensure that you have the following installed on your system:
+
+- [Git](https://git-scm.com/book/en/v2/Getting-Started-Installing-Git)
+- [Poetry](https://python-poetry.org/docs/#installation)
+- [Conda](https://docs.conda.io/projects/conda/en/latest/user-guide/install/index.html) (miniconda or anaconda)
+
+## Clone the Repository
+
+```bash
+git clone https://github.com/codecentric-oss/niceml.git
+cd niceml
+```
+
+## Create a Conda Environment
+
+We recommend using conda to manage your Python environment. Navigate to the project directory and create a conda environment:
+
+```bash
+conda create --name your-env-name python=3.x
+conda activate your-env-name
+```
+
+Replace `your-env-name` with your desired environment name and `3.x` with a Python version above 3.8 and below 3.11.
```

Comment:
If you are working with a Apple Silicon we recommend using python 3.8

Reply:
Added the information for Apple Silicon